pkg/dmesg/watcher.go

```diff
@@ -0,0 +1,115 @@
+package dmesg
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/leptonai/gpud/log"
+	"github.com/leptonai/gpud/pkg/process"
+)
+
+type LogLine struct {
+	Timestamp time.Time
```

Comment:
doesnt dmesg have defined level?

Reply:
```
kern  :warn  : 2025-01-21T04:41:44,285147+00:00 NVRM: Xid (PCI:0000:38:00): 13, pid='<unknown>', name=<unknown>, Graphics Exception: ESR 0x556730=0x117000e 0x556734=0x24 0x556728=0xf81eb60 0x55672c=0x1174
kern  :warn  : 2025-01-21T04:41:44,287197+00:00 NVRM: Xid (PCI:0000:38:00): 43, pid=2924364, name=pt_main_thread, Ch 00000008
```

Let me use `dmesg --decode --time-format=iso --nopager` to get facility + level

Reply:
Now supports facility + level